pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -345,8 +346,14 @@ func applyCounterOrListPolicy(c *autoscalingv1.CounterPolicy, l *autoscalingv1.L
 			return scaleLimited(scale, f, gameServerLister, nodeCounts, key, isCounter, replicas,
 				capacity, aggCapacity, minCapacity, maxCapacity)
 		}
-		return scaleDown(f, gameServerLister, nodeCounts, key, isCounter, replicas, aggCount,
-			aggCapacity, minCapacity, buffer)
+		if replicas > 1 {
+			result, isLimited, err := scaleDown(f, gameServerLister, nodeCounts, key, isCounter, replicas, aggCount,
+				aggCapacity, minCapacity, buffer)
+			if result <= 1 {
+				return 1, true, nil
+			}
+			return result, isLimited, err
```

Comment:
```suggestion
	  if result <= 1 {
		  return 1, true, nil
	  }

    return scaleDown(f, gameServerLister, nodeCounts, key, isCounter, replicas, aggCount,
				aggCapacity, minCapacity, buffer)
```

@igooch is that what you meant? (Also just a bit neater)

Reply:
Thank you for the clarification. I just wanted to confirm one detail, did you mean to use replicas in the condition?

```
if replicas <= 1 {
		  return 1, true, nil
	  }
    return scaleDown(f, gameServerLister, nodeCounts, key, isCounter, replicas, aggCount,
				aggCapacity, minCapacity, buffer)
```

Reply:
We may need some more time to go over this logic. The current number of replicas is not directly relevant to Counts and Lists as the desired replicas are not calculated based on current replicas. 

Also the logic if 
```
result <= 1 {
    return 1, true, nil
}
```
will also not be true in all cases. There are some cases where 1 is the desired number of replicas and it was not limited.

Reply:
oooh, I see what you saying.

It probably should be:

```go
result < 1 {
    return 1, true, nil
}
```

?

Reply:
Admitedly, I should sit down and review this code again, because I think I might have taken too cursory a glance.

Should it actually be `result < b.MinReplicas` (are we checking Mn/Max values right now I could see it?)